### PR TITLE
Create GridexTasksTutorials.md

### PR DESCRIPTION
**Pull request by navillanueva**
first commit

# Pull request checklist

Use the following template to make sure your PR fits the ConsenSys documentation standard.

## Before creating the PR

Make sure that:

- [ ] You've read the [contribution guidelines](https://docs-template.consensys.net/getting-started/style-guide).
- [ ] You've previewed your changes locally.

## Describe the change

Updated the links that went to blockscout and changed them to the goerli.lineascan.build

## Issue fixed

No current issue exists

## Impacted parts

For content changes:

- [ x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] Build and QA tools
- [ ] Node dependencies and JavaScript
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [ ] You've fixed any issues raised by the tests.
- [ ] You've previewed your changes on Vercel below.

### CODE REVIEW

**Comment by navillanueva** [comment]
Pull Request Turotial for Tasks in Defi Week for Gridex Protocol

**Comment by emilyJLin95** [comment]
@navillanueva 
Can you fix these linting errors: https://github.com/ConsenSys/doc.zk-evm/actions/runs/5356783757/jobs/9717644721?pr=176

Add words to `project-words.txt` and you can run `npm run lint` to check it works

**Comment by navillanueva** [comment]
@emilyJLin95 
I added the words, looks like its working :)

File: GridexTasksTutorials.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Gridex Protocol
```
Comment (emilyJLin95) [inline GridexTasksTutorials.md:1]: @navillanueva 
```suggestion
---
title: Use Gridex Protocol
sidebar_position: 1
---

# Gridex Protocol
```

**Comment by emilyJLin95** [comment]
@navillanueva awesome! made one last suggestion

**Review by emilyJLin95 — APPROVED** [review]
LGTM

**Pull request merged**